modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/LinkOrButton.js

```diff
@@ -47,6 +47,7 @@ const LinkOrButton = ({
 				})}
 				disabled={disabled}
 				href={href}
+				role="button"
```

Comment:
@carloslancha do we need to use this both cases? <wrapper> can be a ClayLink too

Reply:
What do you mean?

Reply:
I mean that when `<Wrapper>` is `<ClayButton>` the `role="button"` looks redundant. There's some literature about this topic, all I've checked says this is not necessary. rocketvalidator includes a [check for this](https://rocketvalidator.com/html-validation/the-button-role-is-unnecessary-for-element-button), and latest aria spec seems [reluctant to this practise](https://www.w3.org/TR/html-aria/#avoid-specifying-redundant-roles)

So we likely need to include the role only if we're using ClayLink:
```suggestion
  const isLink = href && !disabled;
  const Wrapper = isLink ? ClayLink : ClayButton;
  ...  
  role={isLink ? "button" : undefined}
```
@carloslancha if you're ok pls make the changes and resend

Reply:
We can't fix it this way since this would affect to all the usages of `LinkOrButton`, and I don't think we want all the links to have `role="button"`, right?

Reply:
yep, maybe we need to assert this is the case for all usages of LinkOrButton. Otherwise that logic shall go to the calling component (MT in this case)